target/linux/ramips/dts/mt7621_humax_e10.dts

```diff
@@ -0,0 +1,153 @@
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/mtd/partitions/uimage.h>
+
+/ {
+	compatible = "humax,e10", "mediatek,mt7621-soc";
+	model = "HUMAX QUANTUM E10";
+
+	aliases {
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+		label-mac-device = &wan;
+	};
+	
+	gpio-export {
+		compatible = "gpio-export";
+
+		gpio_usb_power {
+                        gpio-export,name = "humax:power:usb";
+                        gpio-export,output = <1>;
+                        gpios = <&gpio 8 GPIO_ACTIVE_HIGH>;
+                };
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "humax:green:power";
+			gpios = <&gpio 6 GPIO_ACTIVE_HIGH>;
+		};
+
+		status {
+			label = "humax:green:status";
+			gpios = <&gpio 13 GPIO_ACTIVE_HIGH>;
+		};
+
+		wlan {
+			label = "humax:green:wlan";
+			gpios = <&gpio 7 GPIO_ACTIVE_HIGH>;
+			linux,default-trigger = "phy1radio";
+		};
+
+		wps {
+			label = "humax:green:wps";
+			gpios = <&gpio 12 GPIO_ACTIVE_HIGH>;
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		wps_reset {
+			label = "wps_reset";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <50000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x0 0x30000>;
+				read-only;
+			};
+
+			partition@30000 {
+				label = "u-boot-env";
+				reg = <0x30000 0x10000>;
+				read-only;
+			};
+
+			factory: partition@40000 {
+				label = "factory";
+				reg = <0x40000 0x30000>;
+				read-only;
+			};
+
+                        partition@70000 {
+				compatible = "openwrt,uimage", "denx,uimage";
+                                openwrt,offset = <FW_EDIMAX_OFFSET>;
+                                openwrt,partition-magic = <FW_MAGIC_EDIMAX>;
+                                label = "firmware";
+                                reg = <0x70000 0xf90000>;
+                        };
+		};
+	};
+};
+
+&pcie {
+	status = "okay";
+};
+
+&pcie0 {
+	wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0000 0 0 0 0>;
+		mediatek,mtd-eeprom = <&factory 0x0000>;
+		ieee80211-freq-limit = <2400000 2500000>;
+	};
+};
+
+&pcie1 {
+	wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0000 0 0 0 0>;
+		mediatek,mtd-eeprom = <&factory 0x8000>;
+		ieee80211-freq-limit = <5000000 6000000>;
+	};
+};
+
+&gmac0 {
+	mtd-mac-address = <&factory 0x10007>;
```

Comment:
Please replace "mtd-mac-address" with "nvmem-cells".

Reply:
Configured, but I don't know how 'nvmem-cells'... @mans0n, It looks ok?

Reply:
Please refer [other dts file](https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/mt7621_iptime_a8004t.dts). (Ctrl-F "nvmem-cells")

Reply:
Corrected!